### PR TITLE
Copyedits for style, security-jwt-build.adoc

### PR DESCRIPTION
**Pull request by rolfedh**
This PR includes copy edits to enhance clarity, improve consistency, and align the content with style guidelines for a better reader experience.

### CODE REVIEW

File: docs/src/main/asciidoc/security-jwt-build.adoc [hunk 1]
```diff
@@ -30,14 +32,15 @@ SmallRye JWT Build provides an API for securing JWT claims using all of these op
...
+
 [source,gradle,role="secondary asciidoc-tabs-target-sync-gradle"]
 .build.gradle
 ----
 implementation("io.quarkus:quarkus-smallrye-jwt-build")
 ----
 
-Note you can use SmallRye JWT Build API without having to create MicroProfile JWT endpoints supported by `quarkus-smallrye-jwt`.
-It can also be excluded from `quarkus-smallrye-jwt` if MP JWT endpoints do not need to generate JWT tokens.
+You can use the SmallRye JWT Build API independently, without needing to create MicroProfile JWT endpoints supported by the `quarkus-smallrye-jwt` extension. Additionally, the API can be excluded from `quarkus-smallrye-jwt` if there is no requirement to generate JWT tokens for MicroProfile JWT endpoints.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-jwt-build.adoc:43]:
```suggestion
You can use the SmallRye JWT Build API independently, without needing to create MicroProfile JWT endpoints supported by the `quarkus-smallrye-jwt` extension.
```
Not related to the PR, but this is an old suggestion which might work if the optimization is needed but more likely will cause side-effects, with the latest devmode support work, and in native, so better remove it

File: docs/src/main/asciidoc/security-jwt-build.adoc [hunk 2]
```diff
@@ -55,57 +58,73 @@ import org.eclipse.microprofile.jwt.JsonWebToken;
...
-Note the `alg` (algorithm) header is set to `RS256` by default. Signing key identifier (`kid` header) does not have to be set if a single JSON Web Key (JWK) containing a `kid` property is used.
+Default behaviors:
+
+- The `alg` (algorithm) header is set to `RS256` by default.
+- You do not have to set signing key identifier (`kid` header) if a single JSON Web Key (JWK) containing a `kid` property is used.
+
+Supported keys and algorithms:
 
-RSA and Elliptic Curve (EC) private keys as well as symmetric secret keys can be used to sign the claims.
-`ES256` and `HS256` are the default algorithms for EC private and symmetric key algorithms respectively.
```
Comment (rolfedh) [inline docs/src/main/asciidoc/security-jwt-build.adoc:108]: @sberyozkin Is there a default algorithm for RSA private keys?

**Comment by rolfedh** [comment]
Hi, @sberyozkin. Please review and comment on this revision. Thank you.

File: docs/src/main/asciidoc/security-jwt-build.adoc (hunk 2 above)
Comment (sberyozkin) [reply docs/src/main/asciidoc/security-jwt-build.adoc:108]: @rolfedh `RS256` is the default one for RSA keys

Comment (rolfedh) [reply docs/src/main/asciidoc/security-jwt-build.adoc:108]: Thank you. Updated.

File: docs/src/main/asciidoc/security-jwt-build.adoc [hunk 3]
```diff
@@ -55,117 +58,142 @@ import org.eclipse.microprofile.jwt.JsonWebToken;
...
+- The `alg` (algorithm) header is set to `RS256` by default.
+- You do not have to set a signing key identifier (`kid` header) if a single JSON Web Key (JWK) containing a `kid` property is used.
 
-RSA and Elliptic Curve (EC) private keys as well as symmetric secret keys can be used to sign the claims.
-`ES256` and `HS256` are the default algorithms for EC private and symmetric key algorithms respectively.
+Supported keys and algorithms:
 
-You can customize the signature algorithm, for example:
+- To sign the claims, you can use RSA private keys, Elliptic Curve (EC) private keys, and symmetric secret keys.
+- `ES256` is the default algorithm for RSA and EC private keys
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-jwt-build.adoc:126]: @rolfedh This is not correct, I mentioned `RS256` for RSA keys. The first line should read `RS256 is the default algorithm for RSA private keys`

Comment (sberyozkin) [inline docs/src/main/asciidoc/security-jwt-build.adoc:126]:
```suggestion
- `ES256` is the default algorithm for EC private keys
```

Comment (rolfedh) [reply docs/src/main/asciidoc/security-jwt-build.adoc:126]: Thank you. Sorry I misread your initial message.

File: docs/src/main/asciidoc/security-jwt-build.adoc [hunk 4]
```diff
@@ -55,117 +58,143 @@ import org.eclipse.microprofile.jwt.JsonWebToken;
...
+
+- The `alg` (key management algorithm) header defaults to `RSA-OAEP`.
+- The `enc` (content encryption) header defaults to `A256GCM`.
 
-RSA and Elliptic Curve (EC) public keys as well as symmetric secret keys can be used to encrypt the claims.
-`ECDH-ES` and `A256KW` are the default algorithms for EC public and symmetric key encryption algorithms respectively.
+Supported keys and algorithms:
+
+- You can use RSA public keys, Elliptic Curve (EC) public keys, and symmetric secret keys, to encrypt the claims.
+- `ECDH-ES` is the default algorithm for the RSA and EC public keys.
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply docs/src/main/asciidoc/security-jwt-build.adoc]: Updated. Thanks.

File: docs/src/main/asciidoc/security-jwt-build.adoc [hunk 5]
```diff
@@ -55,117 +58,144 @@ import org.eclipse.microprofile.jwt.JsonWebToken;
...
 
-Note the `alg` (key management algorithm) header is set to `RSA-OAEP` and the `enc` (content encryption header) is set to `A256GCM` by default.
+Default behaviors:
+
+- The `alg` (key management algorithm) header defaults to `RSA-OAEP`.
+- The `enc` (content encryption) header defaults to `A256GCM`.
 
-RSA and Elliptic Curve (EC) public keys as well as symmetric secret keys can be used to encrypt the claims.
-`ECDH-ES` and `A256KW` are the default algorithms for EC public and symmetric key encryption algorithms respectively.
+Supported keys and algorithms:
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-jwt-build.adoc:186]: Sorry @rolfedh It reads not right, I'm sorry, I'm just trying to finish my own PR and getting a bit distracted, let's try to finalize it

File: docs/src/main/asciidoc/security-jwt-build.adoc [hunk 6]
```diff
@@ -55,117 +58,144 @@ import org.eclipse.microprofile.jwt.JsonWebToken;
...
+
+- You can use RSA public keys, Elliptic Curve (EC) public keys, and symmetric secret keys, to encrypt the claims.
+- `RSA-OAEP` is the default algorithm for the RSA.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-jwt-build.adoc:189]:
```suggestion
- `RSA-OAEP` is the default algorithm for the RSA public key encryption algorithms.
```

File: docs/src/main/asciidoc/security-jwt-build.adoc [hunk 7]
```diff
@@ -55,117 +58,144 @@ import org.eclipse.microprofile.jwt.JsonWebToken;
...
+- `ECDH-ES` is the default algorithm for the EC public keys.
+- `A256KW` is the default algorithm for the symmetric secret keys.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-jwt-build.adoc:191]:
```suggestion
- `A256KW` is the default algorithm for the symmetric key encryption algorithms.
```

This it equivalent now to what is said in the replaced sentence

File: docs/src/main/asciidoc/security-jwt-build.adoc [hunk 8]
```diff
@@ -55,117 +58,144 @@ import org.eclipse.microprofile.jwt.JsonWebToken;
...
+- The `alg` (algorithm) header is set to `RS256` by default.
+- You do not have to set a signing key identifier (`kid` header) if a single JSON Web Key (JWK) containing a `kid` property is used.
 
-RSA and Elliptic Curve (EC) private keys as well as symmetric secret keys can be used to sign the claims.
-`ES256` and `HS256` are the default algorithms for EC private and symmetric key algorithms respectively.
+Supported keys and algorithms:
 
-You can customize the signature algorithm, for example:
+- To sign the claims, you can use RSA private keys, Elliptic Curve (EC) private keys, and symmetric secret keys.
+- `RS256` is the default algorithm for RSA private keys.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-jwt-build.adoc:126]:
```suggestion
- `RS256` is the default algorithm for RSA private key signature algorithms.
```

File: docs/src/main/asciidoc/security-jwt-build.adoc [hunk 9]
```diff
@@ -55,117 +58,144 @@ import org.eclipse.microprofile.jwt.JsonWebToken;
...
+- `ES256` is the default algorithm for EC private keys.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-jwt-build.adoc:127]:
```suggestion
- `ES256` is the default algorithm for EC private key signature algorithms.
```

File: docs/src/main/asciidoc/security-jwt-build.adoc [hunk 10]
```diff
@@ -55,117 +58,144 @@ import org.eclipse.microprofile.jwt.JsonWebToken;
...
+- `HS256` is the default algorithm for symmetric secret keys.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-jwt-build.adoc:128]:
```suggestion
- `HS256` is the default algorithm for symmetric key signature algorithms.
```

**Comment by sberyozkin** [comment]
Sorry @rolfedh, I don't mean to be too pedantic, these are last proposed updates,  it is just saying a `algorithm for the key` does not read well, the key is just a blob, and it can be used for different purposes...

File: docs/src/main/asciidoc/security-jwt-build.adoc (hunk 5 above)
Comment (rolfedh) [reply docs/src/main/asciidoc/security-jwt-build.adoc:186]: Thank you. Updated per our Slack conversation. Please let me know if you have any further suggestions or feedback.

File: docs/src/main/asciidoc/security-jwt-build.adoc (hunk 6 above)
Comment (rolfedh) [reply docs/src/main/asciidoc/security-jwt-build.adoc]: Thank you. Updated per our Slack conversation. Please let me know if you have any further suggestions or feedback.

File: docs/src/main/asciidoc/security-jwt-build.adoc (hunk 7 above)
Comment (rolfedh) [reply docs/src/main/asciidoc/security-jwt-build.adoc]: Thank you. Updated per our Slack conversation. Please let me know if you have any further suggestions or feedback.

File: docs/src/main/asciidoc/security-jwt-build.adoc (hunk 8 above)
Comment (rolfedh) [reply docs/src/main/asciidoc/security-jwt-build.adoc]: Thank you. Updated per our Slack conversation. Please let me know if you have any further suggestions or feedback.

File: docs/src/main/asciidoc/security-jwt-build.adoc (hunk 10 above)
Comment (rolfedh) [reply docs/src/main/asciidoc/security-jwt-build.adoc]: Thank you. Updated per our Slack conversation. Please let me know if you have any further suggestions or feedback.

**Comment by sberyozkin** [comment]
Thanks very much @rolfedh, good to see this text being so nicely polished, thanks for that, my suggestions were very minor ones...

**Review by sberyozkin — APPROVED** [review]

**Pull request merged**